src/containers/tutor-home-page/general-info-step/SelectGroup.tsx

```diff
@@ -1,30 +1,78 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import { FormControl, InputLabel, Select, MenuItem } from '@mui/material'
 import { useTranslation } from 'react-i18next'
 import { useSelectGroupStyles } from './SelectGroup.styles'
+import { LocationService } from '~/services/location-service'
+interface Country {
+  name: string
+}
 
+interface City {
+  name: string
+}
 const SelectGroup: React.FC = () => {
   const { t } = useTranslation()
   const classes = useSelectGroupStyles()
+  const [countries, setCountries] = useState([])
+  const [cities, setCities] = useState([])
+  const [selectedCountry, setSelectedCountry] = useState('')
+
+  useEffect(() => {
+    const fetchCountries = async () => {
+      try {
+        const response = await LocationService.getCountries()
+        setCountries(response.data as Country[])
+      } catch (error) {
+        console.error('Error fetching countries:', error)
+      }
+    }
+
+    void fetchCountries()
+  }, [])
+
+  useEffect(() => {
+    if (selectedCountry) {
+      const fetchCities = async () => {
+        try {
+          const response = await LocationService.getCities(selectedCountry)
+          setCities(response.data as City[])
+        } catch (error) {
+          console.error('Error fetching cities:', error)
+        }
+      }
+
+      void fetchCities()
+    }
+  }, [selectedCountry])
+
+  const handleCountryChange = (event: {
+    target: { value: React.SetStateAction<string> }
+  }) => {
```

Comment:
```suggestion
  const handleCountryChange = (event: SelectChangeEvent) => {
```

### You can import SelectChangeEvent type from MUI to get full typescript support

**After:** 
![image](https://github.com/Space2Study-UA-1156/Client-01/assets/56509285/1d71d215-968a-4338-8208-27f3527ee20f)

**Before:** 
![image](https://github.com/Space2Study-UA-1156/Client-01/assets/56509285/a1f95343-5852-4fd0-bea0-6c3c94cbd1d7)





![image](https://github.com/Space2Study-UA-1156/Client-01/assets/56509285/5d24515b-199a-4023-a1b0-13da195712c4)